sdk/devcenter/readme.md

```diff
@@ -0,0 +1,55 @@
+# devtestcenter
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Azure.Developer.DevCenter.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
```

Comment:
I think we don't need to add autorest configuration in this readme.md file. You can add this information directly in your [autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/26cc25fb1dc9e191a88d76c5d5b16b2af14efa39/sdk/devcenter/Azure.Developer.DevCenter/src/autorest.md) file. You can refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/src/autorest.md) autorest.md as an example.

This readme.md  file should include basic information about DevCenter. You can refer this [readme.md](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/README.md).